app/v2/notifications/post_notifications.py

```diff
@@ -64,6 +64,8 @@
 
 @v2_notification_blueprint.route('/<notification_type>', methods=['POST'])
 def post_notification(notification_type):  # noqa: C901
+
+    form = None
```

Comment:
```suggestion
    form = {}
```
Lines 98, 107, and 108 fail if this is set to None instead of a dictionary. Line 107 requires a `.get('template_id')`, not `['template_id']`.